src/app/signin/page.tsx

```diff
@@ -0,0 +1,162 @@
+
+      }));
+    } else {
+      // Handle successfull login
+      console.log("Logged in:", email)
+    }
+    
+  };
+
+  return (
+    <section className="login-bg">
+    <div className="login-container">
+      <div className="login-header">
+      <Link
+        href="/"
+        style={{ 
+          textDecoration: 'none'
+        }}>
+        <div className="logo-container">
+          <Image src="/icons/logo.png" alt="logo" width={60} height={60} />
+          <p>Song Sieve </p>
+        </div>
+      </Link>
+      </div>
+
+      <div className="form-container">
+        <h2>Sign In</h2>
+        <form onSubmit={handleSubmit}>
+          <div className="form-group">
+            <input
+              type="email"
+              name="email"
+              className="form-control"
+              placeholder="Email"
+              value={auth.email}
+              onChange={handleInputChange}
+            />
+            <Image
+              src="/icons/email-icon.svg"
+              alt="Email"
+              className="input-icon"
+              width={25}
+              height={25}
+            />
+            <div className="error">{auth.errors.email}</div>
+          </div>
+
+          <div className="form-group">
+            <input
+              type="password"
+              name="password"
+              className="form-control"
+              placeholder="Password"
+              value={auth.password}
+              onChange={handleInputChange}
+            />
+            <Image
+              src="/icons/lock-icon.svg"
+              alt="Password"
+              className="input-icon"
+              width={25}
+              height={25}
+            />
+            <div className="error">{auth.errors.password}</div>
+          </div>
+
+          <div className="Forget-password">
+            <p >Forget Password?</p>
+          </div>
+
+          <button type="submit" className="submit-btn"><span>SIGN IN</span></button>
+
+        </form>
+      </div>
+
+
+
+      <div className="connect-container">
+        <p>Be connect with</p>
+
+        <div className="icon-connect">
+          <Image
+            src="/icons/google-icon.svg"
+            alt="Google icon"
+            width={40}
+            height={40}
+            className="icon"
+            style={{
+              filter: 'invert(100%)',
+              border: '1px solid black',
+              borderRadius: '50%',
+              padding:'5px',
+              background: 'white'}} 
+          />
+          <Image
+          src="/icons/github-icon.svg"
+          alt="GitHub icon"
+          width={40}
+          height={40}
+          className="icon"
+          style={{ 
+            filter: 'invert(100%)',
+            border: '1px solid black',
+            borderRadius: '50%',
+            padding: '5px',
+            background: 'white'
+          }} 
+
+
+          />
+        </div>
+        
+        <p>Don’t have an account? <a>Sign up</a></p>
```

Comment:
Here - instead of using <a> tag => please use Link from "next/naviagtion"
https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating